src/components/Garden/ModalFlows/CreateProposalScreens/CreateProposalScreens.js

```diff
@@ -7,6 +7,7 @@ import { useAgreement } from '@hooks/useAgreement'
 import { useWallet } from '@providers/Wallet'
 import useActions from '@hooks/useActions'
 import { useStakingState } from '@providers/Staking'
+import confetti from 'canvas-confetti'
```

Comment:
Let´s move this import to line 2.

We usually place external libraries imports first

Reply:
Gotcha! That info need go to some CONTRIBUTING or DEV GUIDE?

Reply:
Yes, we use the following order for imports: `libraries > componentes > providers > hooks > utils > assets`.

Reply:
Wow! Yah that definity need put on guide. I'll compile that and provide some examples.
I'll make it on some GIST to not mess up with review on that PR.

Or make new PR just to GUIDE

Reply:
Yeah that makes more sense!